packages/manager/apps/ips/src/pages/listing/components/DatagridCells/datagridCellStyles.ts

```diff
@@ -0,0 +1,18 @@
+export const datagridCellStyle = {
+  iconDefault: {
+    color: 'var(--ods-color-primary-700)',
+    cursor: 'pointer',
+  },
+  iconWarning: {
+    color: 'var(--ods-color-warning-700)',
+    cursor: 'pointer',
+  },
+  iconSuccess: {
+    color: 'var(--ods-color-success-700)',
+    cursor: 'pointer',
+  },
+  iconDisable: {
+    color: 'var(--ods-color-neutral-500)',
+    cursor: 'pointer',
+  },
+};
```

Comment:
Can we add this inside tailwind config file ? And we can use it as a custom utlis inside tsx file. If we move this on to tailwind file, we can create a consistent and scalable design and avoids importing in each file. Thoughts ?